pandas/tests/tools/test_to_datetime.py

```diff
@@ -2985,30 +2898,6 @@ def test_to_datetime_cache_coerce_50_lines_outofbounds(series_length):
         to_datetime(s, errors="raise", utc=True)
 
 
-@pytest.mark.parametrize(
-    "arg",
-    [
-        ["1724-12-20 20:20:20+00:00", "2022-01-01 00:00:00"],
```

Comment:
since https://github.com/pandas-dev/pandas/pull/49333, the ISO8601 fastpath actually respects the given format, so this would raise

Hence, am removing the test